server/lib/template/partials/enterpriseDetails.handlebars

```diff
@@ -1,34 +1,36 @@
 <!-- this partial requires binding the variable 'metadata' -->
 
-{{#if metadata.enterprise.logoDataURI}}
-  <div class="col-xs-1">
-    <img src="{{metadata.enterprise.logoDataURI}}" width="65" height="65" alt="">
-  </div>
-{{/if}}
+<div class="col-xs-6" style="display: flex; flex-direction: row; justify-content: start;">
```

Comment:
I am okay with flex styles here since we control the chrome browser that renders this template. 👍